src/connect.js

```diff
@@ -76,6 +82,12 @@ function connect(connectOptions, callback) {
         ethereumConnectionInfo.contracts = ethereumConnectionInfo.contracts || contracts.addresses[DEFAULT_NETWORK_ID];
         self.api = api.generateContractApi(ethereumConnectionInfo.abi.functions);
         self.rpc = rpcInterface.createRpcInterface(ethereumConnectionInfo.rpc);
+        ethereumConnectionInfo.rpc.getTransport().addReconnectListener(function () {
+          rpcInterface.emit("reconnect");
```

Comment:
I don't think `rpcInterface` has an `emit` method -- should this be `augurNode.emit` instead?  (Same comment for the disconnect listener callback.)

Reply:
rpcInterface is the ethrpc interface, I mixin EventEmitter there as well.